clients/client-python/gravitino/api/supports_schemas.py

```diff
@@ -25,7 +25,7 @@ class SupportsSchemas(ABC):
     """
 
     @abstractmethod
-    def list_schemas(self, namespace: Namespace) -> List[NameIdentifier]:
```

Comment:
In the new API, this method doesn't have any input parameter (like the namespace). So the namespace information should come from the BaseSchemaCatalog itself. But now the BaseSchemaCatalog object doesn't hold the namespace information, so you need to pass-in that information in the init method. You can refer to the Java change at commit 2687f1aa1819fe301151dded82563911875ddf62

Reply:
the interface not changed?